Extensions/PrimitiveDrawing/shapepainterruntimeobject.ts

```diff
@@ -19,12 +19,12 @@ namespace gdjs {
     fillColor: RGBColor;
     /** The color (in RGB format) of the outline of the painted shape */
     outlineColor: RGBColor;
-    /** The opacity of the inner part of the painted shape */
-    fillOpacity: number;
-    /** The opacity of the outline of the painted shape */
-    outlineOpacity: number;
+    /** The opacity of the inner part of the painted shape, form 0 to 255 */
+    fillOpacity: integer;
+    /** The opacity of the outline of the painted shape, form 0 to 255 */
+    outlineOpacity: integer;
```

Comment:
```suggestion
    /** The opacity of the inner part of the painted shape, from 0 to 255 */
    fillOpacity: float;
    /** The opacity of the outline of the painted shape, from 0 to 255 */
    outlineOpacity: float;
```

Typo + even if it's usually set to an integer, the opacity can (and will) be a float (when it's progressively updated for example)

Reply:
As the implementation uses it more like an integer at this time, I find it less confusing.
Can we let the person that will change it to 0.0 to 1.0 make it float?

Reply:
Well the thing is that it's not an error to have a floating point value (like 254.5), so we should in theory have a float here?